### ISSUE TITLE
EU taxes from 1st of july 2021

### ISSUE THREAD

**Issue by oblaum** [woocommerce/woocommerce#30133]
hello,

as the EU tax handling changes starting from 1st of july with indidivual taxes for each country.
i need to adjust a customers shop. here the customer wants to use the pricemodell to have 
the same end price to all customers (regardless of 19%, 20% or 21% taxes).

this works great with this function:

https://github.com/woocommerce/woocommerce/wiki/How-Taxes-Work-in-WooCommerce#prices-including-tax---experimental-behavior

BUT: 

for EU countries like switzerland (CH), liechtenstein (LI) or norway there should be no taxes applied (0% tax).
im my case i use prices incl. taxes in the products. 

it maybe a woocommerce bug.

so i found this function – this works e.g. for CH but not for the EU countries.

```
add_action( 'woocommerce_cart_calculate_fees', 'same_price_for_eu' );
function same_price_for_eu(){

   $countries = new WC_Countries();

   $eu_countries = $countries->get_european_union_countries();

   if ( in_array( WC()->customer->get_shipping_country(), $eu_countries) ){
  add_filter( 'woocommerce_adjust_non_base_location_prices', '__return_false');

   } else {
   remove_filter( 'woocommerce_adjust_non_base_location_prices', '__return_false');

   }
}
```

in this function maybe in the add action something is missing, as the tax modell is right shown via geop ip shipping address,
but the price including tax is wrong.

**Prerequisites (mark completed items with an [x]):**
- [x] I have have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**

price incl. tax is not shown propper in catalog, product page and cart for non EU countries.

**Expected behavior**
show the right tax price

**Actual behavior**
price is shown only in the EU countries right.

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:
1. Go to https://dev.rodel.at/en/kathrein/family-sledge-two-seater-for-1-2-persons/ 
2. Click on add do cart
3. Go to Cart https://dev.rodel.at/en/cart-2/ 
4. Change Country to Switerland or Germany
5. price is wrong on  https://dev.rodel.at/en/ and https://dev.rodel.at/en/kathrein/family-sledge-two-seater-for-1-2-persons/  and https://dev.rodel.at/en/cart-2/ 

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
We use the [WooCommerce System Status Report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/) to help us evaluate the issue. 
Without this report we won't be able to fully evaluate this issue.
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://dev.rodel.at
Site address (URL): https://dev.rodel.at
WC Version: 5.4.1
REST API Version: ✔ 5.4.1
WC Blocks Version: ✔ 5.1.0
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 2.3.1
Log Directory Writable: ✔
WP Version: 5.7.2
WP Multisite: –
WP Memory Limit: 16 GB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.3.28
PHP Post Max Size: 20 MB
PHP Time Limit: 240
PHP Max Input Vars: 2000
cURL Version: 7.64.0
OpenSSL/1.1.1d

SUHOSIN Installed: –
MySQL Version: 5.7.34-2
Max Upload Size: 20 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 5.4.1
WC Database Prefix: kat_
Total Database Size: 1278.09MB
Database Data Size: 727.15MB
Database Index Size: 550.94MB
kat_woocommerce_sessions: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
kat_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
kat_woocommerce_order_items: Data: 6.52MB + Index: 2.52MB + Engine InnoDB
kat_woocommerce_order_itemmeta: Data: 52.59MB + Index: 58.17MB + Engine InnoDB
kat_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
kat_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_actionscheduler_actions: Data: 0.16MB + Index: 0.20MB + Engine InnoDB
kat_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_actionscheduler_logs: Data: 0.14MB + Index: 0.09MB + Engine InnoDB
kat_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_comments: Data: 11.52MB + Index: 12.09MB + Engine InnoDB
kat_duplicator_pro_entities: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_duplicator_pro_packages: Data: 2.02MB + Index: 0.02MB + Engine InnoDB
kat_gonzales_disabled: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_gonzales_enabled: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_gonzales_p_disabled: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_gonzales_p_enabled: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_icl_content_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_icl_core_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_icl_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_icl_languages: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_icl_languages_translations: Data: 0.19MB + Index: 0.11MB + Engine InnoDB
kat_icl_locale_map: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_icl_message_status: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_icl_mo_files_domains: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_icl_node: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_icl_reminders: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_icl_strings: Data: 6.52MB + Index: 11.06MB + Engine InnoDB
kat_icl_string_batches: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_icl_string_packages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_icl_string_pages: Data: 138.67MB + Index: 194.47MB + Engine InnoDB
kat_icl_string_positions: Data: 0.30MB + Index: 0.09MB + Engine InnoDB
kat_icl_string_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_icl_string_translations: Data: 5.52MB + Index: 4.03MB + Engine InnoDB
kat_icl_string_urls: Data: 0.28MB + Index: 0.33MB + Engine InnoDB
kat_icl_translate: Data: 1.52MB + Index: 0.14MB + Engine InnoDB
kat_icl_translate_job: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_icl_translations: Data: 1.52MB + Index: 1.47MB + Engine InnoDB
kat_icl_translation_batches: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_icl_translation_downloads: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_icl_translation_status: Data: 2.52MB + Index: 0.09MB + Engine InnoDB
kat_iconic_was_fees: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_imagify_files: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
kat_imagify_folders: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_itsec_distributed_storage: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_itsec_fingerprints: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_itsec_geolocation_cache: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_itsec_lockouts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
kat_itsec_logs: Data: 73.09MB + Index: 5.67MB + Engine InnoDB
kat_itsec_temp: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
kat_jckpc_inventory: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_magic360_store: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
kat_magictoolbox_store: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_ms_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_options: Data: 9.13MB + Index: 0.25MB + Engine InnoDB
kat_postmeta: Data: 215.77MB + Index: 115.31MB + Engine InnoDB
kat_posts: Data: 9.52MB + Index: 8.06MB + Engine InnoDB
kat_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_social_users: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_storeabill_documentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_storeabill_documents: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
kat_storeabill_document_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_storeabill_document_items: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
kat_storeabill_document_noticemeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_storeabill_document_notices: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_storeabill_journals: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_termmeta: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
kat_terms: Data: 0.16MB + Index: 0.19MB + Engine InnoDB
kat_term_relationships: Data: 0.30MB + Index: 0.23MB + Engine InnoDB
kat_term_taxonomy: Data: 0.17MB + Index: 0.19MB + Engine InnoDB
kat_tm_taskmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_tm_tasks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_usermeta: Data: 12.52MB + Index: 15.03MB + Engine InnoDB
kat_users: Data: 1.52MB + Index: 0.55MB + Engine InnoDB
kat_wcpdf_credit_note_number: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_wcpdf_invoice_number: Data: 0.31MB + Index: 0.00MB + Engine InnoDB
kat_wcpdf_packing_slip_number: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
kat_wcpdf_proforma_number: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
kat_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
kat_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_wc_product_meta_lookup: Data: 0.33MB + Index: 0.53MB + Engine InnoDB
kat_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_woocommerce_gzd_dhl_im_products: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_gzd_dhl_im_product_services: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_gzd_packaging: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_woocommerce_gzd_packagingmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_gzd_shipmentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_gzd_shipments: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
kat_woocommerce_gzd_shipment_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_gzd_shipment_items: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
kat_woocommerce_gzd_shipment_labelmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_gzd_shipment_labels: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_gzd_shipping_provider: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_woocommerce_gzd_shipping_providermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_woocommerce_oppcw_cc: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
kat_woocommerce_oppcw_documents: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_woocommerce_oppcw_ecc: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_woocommerce_oppcw_schema_version: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_woocommerce_oppcw_storage: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
kat_woocommerce_oppcw_transactions: Data: 21.38MB + Index: 0.00MB + Engine InnoDB
kat_wpr_rucss_resources: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
kat_wpr_rucss_used_css: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
kat_yith_wcwl: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_yith_wcwl_lists: Data: 0.16MB + Index: 0.36MB + Engine InnoDB
kat_yoast_indexable: Data: 4.52MB + Index: 1.00MB + Engine InnoDB
kat_yoast_indexable_hierarchy: Data: 0.08MB + Index: 0.05MB + Engine InnoDB
kat_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
kat_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
kat_yoast_prominent_words: Data: 0.17MB + Index: 0.22MB + Engine InnoDB
kat_yoast_seo_links: Data: 2.52MB + Index: 3.03MB + Engine InnoDB
kat_yoast_seo_meta: Data: 0.16MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_comments: Data: 4.52MB + Index: 3.78MB + Engine InnoDB
x-bak-23200254__kat_duplicator_pro_entities: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_duplicator_pro_packages: Data: 0.20MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_icl_content_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_icl_core_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_icl_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_icl_languages: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_icl_languages_translations: Data: 0.19MB + Index: 0.11MB + Engine InnoDB
x-bak-23200254__kat_icl_locale_map: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_icl_message_status: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_icl_mo_files_domains: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_icl_node: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_icl_reminders: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_icl_strings: Data: 5.52MB + Index: 6.55MB + Engine InnoDB
x-bak-23200254__kat_icl_string_packages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_icl_string_pages: Data: 14.52MB + Index: 11.52MB + Engine InnoDB
x-bak-23200254__kat_icl_string_positions: Data: 0.28MB + Index: 0.08MB + Engine InnoDB
x-bak-23200254__kat_icl_string_status: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_icl_string_translations: Data: 5.52MB + Index: 2.52MB + Engine InnoDB
x-bak-23200254__kat_icl_string_urls: Data: 0.28MB + Index: 0.33MB + Engine InnoDB
x-bak-23200254__kat_icl_translate: Data: 1.52MB + Index: 0.13MB + Engine InnoDB
x-bak-23200254__kat_icl_translate_job: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_icl_translations: Data: 0.52MB + Index: 1.25MB + Engine InnoDB
x-bak-23200254__kat_icl_translation_batches: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_icl_translation_status: Data: 1.52MB + Index: 0.09MB + Engine InnoDB
x-bak-23200254__kat_iconic_was_fees: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_imagify_files: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
x-bak-23200254__kat_imagify_folders: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_itsec_distributed_storage: Data: 3.52MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_itsec_fingerprints: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_itsec_geolocation_cache: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_itsec_lockouts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
x-bak-23200254__kat_itsec_logs: Data: 9.52MB + Index: 0.59MB + Engine InnoDB
x-bak-23200254__kat_itsec_temp: Data: 0.05MB + Index: 0.06MB + Engine InnoDB
x-bak-23200254__kat_jckpc_inventory: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_magic360_store: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_magictoolbox_store: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_options: Data: 3.52MB + Index: 0.14MB + Engine InnoDB
x-bak-23200254__kat_postmeta: Data: 54.58MB + Index: 50.16MB + Engine InnoDB
x-bak-23200254__kat_posts: Data: 4.52MB + Index: 3.81MB + Engine InnoDB
x-bak-23200254__kat_social_users: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_termmeta: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
x-bak-23200254__kat_terms: Data: 0.16MB + Index: 0.19MB + Engine InnoDB
x-bak-23200254__kat_term_relationships: Data: 0.30MB + Index: 0.23MB + Engine InnoDB
x-bak-23200254__kat_term_taxonomy: Data: 0.16MB + Index: 0.17MB + Engine InnoDB
x-bak-23200254__kat_usermeta: Data: 6.52MB + Index: 8.03MB + Engine InnoDB
x-bak-23200254__kat_users: Data: 0.31MB + Index: 0.30MB + Engine InnoDB
x-bak-23200254__kat_wcpdf_credit_note_number: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_wcpdf_invoice_number: Data: 0.11MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_wcpdf_packing_slip_number: Data: 0.16MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_wcpdf_proforma_number: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_order_itemmeta: Data: 18.55MB + Index: 20.03MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_order_items: Data: 2.52MB + Index: 1.52MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
x-bak-23200254__kat_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
x-bak-23200254__kat_yith_wcwl: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
x-bak-23200254__kat_yith_wcwl_lists: Data: 0.08MB + Index: 0.11MB + Engine InnoDB
x-bak-23200254__kat_yoast_seo_links: Data: 1.52MB + Index: 0.50MB + Engine InnoDB
x-bak-23200254__kat_yoast_seo_meta: Data: 0.08MB + Index: 0.00MB + Engine InnoDB

### Post Type Counts ###

attachment: 4439
blocks: 78
custom_css: 1
customize_changeset: 27
document_template: 4
featured_item: 8
nav_menu_item: 46
oembed_cache: 8
page: 90
post: 2
product: 157
product_variation: 3149
revision: 2
shop_coupon: 24
shop_order: 18627
shop_order_refund: 30
wcam: 3
wccf_product_field: 4
wpcf7_contact_form: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (53) ###

WPML Multilingual CMS: by OnTheGoSystems – 4.4.10
WPML SEO: by OnTheGoSystems – 2.0.1
Actionable Google Analytics: by Tatvic – CC-V3-1.7
Code Snippets: by Code Snippets Pro – 2.14.1
Honeypot for Contact Form 7: by Nocean – 2.1
Contact Form 7 Multilingual: by OnTheGoSystems – 1.0.2
Contact Form 7: by Takayuki Miyoshi – 5.4.1
Cookie Notice & Compliance for GDPR / CCPA: by Hu-manity.co – 2.0.4
Duplicate Page and Post: by Arjun Thakur – 2.6.5
Duplicator Pro: by Snap Creek – 4.0.3
Easy WP SMTP: by wpecommerce
alexanderfoxc – 1.4.6

Favicon by RealFaviconGenerator: by Philippe Bernard – 1.3.20
Gonzales: by Tomasz Dobrzyński – 2.1.2
WooCommerce Attribute Swatches by Iconic: by Iconic  – 1.3.3
WooCommerce Product Configurator by Iconic: by Iconic – 1.4.0
Imagify: by WP Media – 1.9.14
iThemes Security Pro: by iThemes – 6.0.3
Magic 360: by Magic Toolbox – 6.8.45
Multiple Domain Mapping on single site: by Matthias Wagner - FALKEmedia – 1.0.5
One Stop Shop for WooCommerce: by vendidero – 1.0.3
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.5
User Role Editor: by Vladimir Garagulya – 4.59.1
Vendidero Helper: by Vendidero – 1.3.0
WooCommerce Active Payments: by WP Desk – 3.6.6
WooCommerce Admin: by WooCommerce – 2.3.1
Advanced Messages for WooCommerce: by Jeroen Sormani – 1.4.1
WooCommerce Custom Fields: by RightPress – 2.3.4
WooCommerce PayPal Checkout Gateway: by WooCommerce – 2.1.1
Germanized for WooCommerce Pro: by vendidero – 3.2.2
Germanized for WooCommerce: by vendidero – 3.4.8
WooCommerce Multilingual: by OnTheGoSystems – 4.11.5
Pay for Payment for WooCommerce: by Karolína Vyskočilová – 2.0.19
WooCommerce PDF Invoices & Packing Slips: by Ewout Fernhout – 2.9.0
WooCommerce PDF Invoices & Packing Slips Professional: by Ewout Fernhout – 2.8.0
WooCommerce PDF Invoices & Packing Slips Premium Templates: by Ewout Fernhout – 2.11.2
WooCommerce Pretty Emails: by MB Création – 1.8.6
WooCommerce Price Based on Country (Basic): by Oscar Gare – 2.0.21
WooCommerce Sequential Order Numbers Pro: by SkyVerge – 1.11.3
Sofort Banking for WooCommerce: by Sven Wagener – 1.4.5
WooCommerce: by Automattic – 5.4.1
WooCommerce OPPCw: by customweb GmbH – 4.0.23
WooSidebars: by WooCommerce – 1.4.5
Yoast SEO: by Team Yoast – 16.5
WP Downgrade | Specific Core Version: by Reisetiger – 1.2.2
WP Rocket | Settings Access: by WP Rocket Support Team –
WP Rocket: by WP Media – 3.9.0.3
WP Rollback: by Impress.org – 1.7.1
WPML Media: by OnTheGoSystems – 2.6.4
WPML String Translation: by OnTheGoSystems – 3.1.8
WPML Translation Management: by OnTheGoSystems – 2.10.6
WP Overnight Sidekick: by Jeremiah Prummer
Ewout Fernhout
Michael Kluver – 2.3.1

Yoast SEO: WooCommerce: by Team Yoast – 12.0
YITH WooCommerce Wishlist: by YITH – 3.0.23

### Inactive Plugins (1) ###

EU/UK VAT Compliance for WooCommerce (Free): by David Anderson – 1.22.0

### Dropin Plugins (1) ###

advanced-cache.php: advanced-cache.php

### Must Use Plugins (1) ###

Gonzales MU: by Tomasz Dobrzyński – 1.0.1

### Settings ###

API Enabled: –
Force SSL: ✔
Currency: EUR (€)
Currency Position: right_space
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #4 - shop/
Cart: #5 - warenkorb/
Checkout: #6 - kasse/
My account: #7 - mein-konto/
Terms and conditions: #26 - agb/

### Theme ###

Name: Kathrein
Version: 1.0
Author URL: http://www.oblaum.com
Child Theme: ✔
Parent Theme Name: Flatsome
Parent Theme Version: 3.13.3
Parent Theme Author URL: http://www.uxthemes.com/
WooCommerce Support: ✔

### Templates ###

Overrides: flatsome/woocommerce/archive-product.php
flatsome/woocommerce/cart/cart-empty.php
flatsome/woocommerce/cart/cart-shipping.php
flatsome/woocommerce/cart/cart.php
flatsome/woocommerce/checkout/form-checkout.php
flatsome/woocommerce/checkout/form-coupon.php
flatsome/woocommerce/checkout/terms.php
flatsome/woocommerce/checkout/thankyou.php
flatsome/woocommerce/content-product-cat.php
flatsome/woocommerce/content-product.php
flatsome/woocommerce/content-single-product.php
flatsome/woocommerce/content-widget-product.php
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/admin-cancelled-order.php version 2.3.0 is out of date. The core version is 4.1.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/admin-failed-order.php version 2.5.0 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/admin-new-order.php version 2.0.0 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/customer-completed-order.php version 1.6.4 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/customer-invoice.php version 2.2.0 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/customer-new-account.php version 1.6.4 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/customer-note.php version 1.6.4 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/customer-on-hold-order.php version - is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/customer-processing-order.php version 1.6.4 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/customer-refunded-order.php version 2.4.0 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/customer-reset-password.php version 2.0.0 is out of date. The core version is 4.0.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/email-addresses.php version 2.2.0 is out of date. The core version is 3.9.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/email-customer-details.php
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/email-footer.php version 2.0.0 is out of date. The core version is 3.7.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails//emails/themes/email-header-centered.php version 1.0 is out of date. The core version is 4.0.0
/usr/www/users/oneetn/dev_rodel_at/wp-content/plugins/woocommerce-pretty-emails/emails/email-order-items.php version 2.1.2 is out of date. The core version is 3.7.0
flatsome/woocommerce/global/breadcrumb.php
flatsome/woocommerce/global/quantity-input.php
flatsome/woocommerce/global/wrapper-end.php
flatsome/woocommerce/global/wrapper-start.php
flatsome/woocommerce/loop/loop-end.php
flatsome/woocommerce/loop/loop-start.php
flatsome/woocommerce/loop/pagination.php
flatsome/woocommerce/loop/result-count.php
flatsome/woocommerce/loop/sale-flash.php
flatsome/woocommerce/myaccount/form-login.php
flatsome/woocommerce/myaccount/navigation.php
flatsome/woocommerce/notices/error.php
flatsome/woocommerce/notices/notice.php
flatsome/woocommerce/notices/success.php
flatsome/woocommerce/product-searchform.php
flatsome/woocommerce/single-product/photoswipe.php
flatsome/woocommerce/single-product/price.php
flatsome/woocommerce/single-product/product-image.php
flatsome/woocommerce/single-product/product-thumbnails.php
flatsome/woocommerce/single-product/rating.php
flatsome/woocommerce/single-product/related.php
flatsome/woocommerce/single-product/review.php
flatsome/woocommerce/single-product/sale-flash.php
flatsome/woocommerce/single-product/share.php
flatsome/woocommerce/single-product/short-description.php
flatsome/woocommerce/single-product/tabs/tabs.php
flatsome/woocommerce/single-product/title.php
flatsome/woocommerce/single-product/up-sells.php
flatsome/woocommerce/single-product-reviews.php
flatsome/woocommerce/single-product.php
flatsome/woocommerce/content-product_cat.php

Outdated Templates: ❌
					
					
						Learn how to update

### Geolocation debug info ###

Default customer location: geolocation
MaxMind GeoIP database: ✔
MaxMind GeoIP license: ✔
MM_COUNTRY_CODE: ❌
GEOIP_COUNTRY_CODE: ❌
HTTP_CF_IPCOUNTRY: ❌
HTTP_X_COUNTRY_CODE: ❌
HTTP_X_REAL_IP: ❌
HTTP_X_FORWARDED_FOR: ❌
REMOTE_ADDR: 84.130.241.103
Real external IP: 2003:d9:6719:e200:1af:df24:178c:3b5a
WCPBC_USE_REMOTE_ADDR: –
Geolocation Test: ✔

### PBC Settings ###

Version: 2.0.21
Base location: AT
Base currency: EUR
Price Based On: billing
Shipping: –
Test mode: –
Test country: –
Load products price in background: –
Prices entered with tax: ✔
Calculate tax based on: billing
Display prices in the shop: incl

### Action Scheduler ###

Complete: 297
Oldest: 2021-06-11 06:55:13 +0200
Newest: 2021-06-21 12:21:56 +0200

Failed: 7
Oldest: 2021-06-10 15:36:03 +0200
Newest: 2021-06-16 03:23:51 +0200

Pending: 2
Oldest: 2021-06-22 11:56:56 +0200
Newest: 2021-06-22 11:56:56 +0200


### Status report information ###

Generated at: 2021-06-21 13:43:18 +02:00
`
```
</details>

**Comment by rodelgc** [comment]
Hi @oblaum,

Thank you for opening the issue! It requires further feedback from the WooCommerce Core team. I am adding the `needs developer feedback` label to this issue so that the Core team could take a look. 

Please note it may take a few days for them to get to this issue. Thank you for your patience.

**Comment by jonathansadowski** [comment]
Hi @oblaum,

While our goal is to address all the issues reported in this repository, GitHub should be treated as a place to report confirmed bugs only. 

The type of issue you submitted looks like a custom code related question. Due to the number of bug reports being submitted, we are prioritizing work on them and are not able to attend to custom code questions. 

The best place to get help further is in the [WooCommerce Community Slack](https://woocommerce.com/community-slack/) in the `#developers` channel. 

If you confirm the bug, please open a new issue and provide us with clear steps to reproduce it.

I am closing this issue for now. If I missed something, please leave a comment and we will take a second look.
